egi_notebooks_hub/d4science.py

```diff
@@ -353,6 +353,15 @@ class D4ScienceSpawner(KubeSpawner):
                 }
             """,
     )
+    extra_profiles = List(
+        [],
+        config=True,
+        help="""Extra profiles to add to user options independenlty of the configuration
```

Comment:
```suggestion
        help="""Extra profiles to add to user options independently of the configuration
```

Reply:
I need to configure my spell checker in vim :)